File: uri/1 - Basics/1060/Main.java
import java.util.*;
class Main{
    public static void main(String args[]){
        Scanner scanner = new Scanner(System.in);

        double a = scanner.nextDouble();
        double b = scanner.nextDouble();
        double c = scanner.nextDouble();
        double d = scanner.nextDouble();
        double e = scanner.nextDouble();
        double f = scanner.nextDouble();

        int num = 0;

        if(a > 0.0) num++;
        if(b > 0.0) num++;
        if(c > 0.0) num++;
        if(d > 0.0) num++;
        if(e > 0.0) num++;
        if(f > 0.0) num++;
        
        System.out.printf("%d valores positivos\n", num);
             
    }
}

File: uri/1 - Basics/1060/Main.go
package main

import (
	"fmt"
)

func main() {
	var a, b, c, d, e, f float32
	num := 0

	fmt.Scanf("%f", &a)
	fmt.Scanf("%f", &b)
	fmt.Scanf("%f", &c)
	fmt.Scanf("%f", &d)
	fmt.Scanf("%f", &e)
	fmt.Scanf("%f", &f)

	if a > 0.0 {
		num++
	}
	if b > 0.0 {
		num++
	}
	if c > 0.0 {
		num++
	}
	if d > 0.0 {
		num++
	}
	if e > 0.0 {
		num++
	}
	if f > 0.0 {
		num++
	}

	fmt.Printf("%d valores positivos\n", num)

}
